src/Link/Link.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { useCallback } from 'react'
+import DefaultLink from './DefaultLink'
+import { WithCoords, ChartNode, LinkProps } from '../types'
+
+interface PropTypes {
```

Comment:
You should reuse the public props since it should be the same (except Component) : use extends